app/src/main/res/values/plurals.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
Let's keep this file but point to strings in strings.xml. If it's allowed, this should be marked as `translatable=false`.

e.g. `<item quantity="zero">@string/no_items</item>`

Reply:
Done!